README.md

```diff
@@ -80,7 +80,7 @@ using SixLabors.ImageSharp.PixelFormats;
 
 // Image.Load(string path) is a shortcut for our default type. 
 // Other pixel formats use Image.Load<TPixel>(string path))
-using (Image<Rgba32> image = Image.Load("foo.jpg"))
+using (Image<Rgba32> image = Image.Load<Rgba32>("foo.jpg"))
```

Comment:
We'd prefer to see (and encourage the usage of) the new, non-generic `Image` class here.

Reply:
Thanks. Makes sense. Updated